.github/workflows/update-and-publish-helm-chart.yml

```diff
@@ -0,0 +1,44 @@
+name: update-and-publish-cromwell-helm-chart
+on:
+  push:
+    branches:
+      - 'main'
```

Comment:
Just to confirm... when this action pushes to main, it doesn't recursively re-trigger itself?

Reply:
That's a good question. I think that it shouldn't recursively re-trigger itself, but I will confirm this statement

Reply:
I am still looking into this. So far Google search hasn't provided me with what I am looking for...

Reply:
It seems that if we use [GITHUB_TOKEN to push things to the repository](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#using-the-github_token-in-a-workflow), it will not trigger recursive workflows runs. Maybe we can use that instead of `BROADBOT_GITHUB_TOKEN`? Or are they one and the same?

Reply:
On a subsequent ticket I set this up intentionally as a test and I can confirm that it DOES recursively re-trigger itself.

Reply:
<img width="971" alt="Screen Shot 2022-08-03 at 7 13 26 PM" src="https://user-images.githubusercontent.com/1087943/182728252-50450237-5d94-4bbb-a32d-29412ff865be.png">

Reply:
And,
```
jobs:
  update-and-publish:
    if: ( !startsWith(github.event.head_commit.message, 'Auto update cromwell-helm chart version') )
```
successfully stops it.